src/smbc/components/_footer.scss

```
@@ -0,0 +1,38 @@
+@import '../settings/all';
+@import '../helpers/all';
+
+.smbc-footer {
+  @extend .govuk-footer; // sass-lint:disable-line placeholder-in-extend
+
+  border-top-width: 4px;
+}
+
+.smbc-footer__logo { // sass-lint:disable-line class-name-format
+  @extend .govuk-footer__copyright-logo; // sass-lint:disable-line placeholder-in-extend
+  @include govuk-media-query($from: mobile) {
+    background-image: url('assets/images/smbc_footer_mobile_logo_180x37.png');
```

We should use the built in function 'govuk-image-url' to load in assets.
```suggestion
    background-image: govuk-image-url('smbc_footer_mobile_logo_180x37.png');
```

---

- `url` has now been changed with `govuk-images-url`
- created an assets file within settings to store image location variables, now using this within footer and header